modulemd/tests/ModulemdTests/moduleindex.py

```diff
@@ -42,15 +42,18 @@ def test_constructors(self):
     def test_read(self):
         idx = ModuleIndex.new()
 
-        with open(path.join(self.source_root, "spec.v1.yaml"), "r") as v1:
+        with open(
+            path.join(self.source_root, "yaml_specs/modulemd_stream_v1.yaml"),
+            "r",
+        ) as v1:
             res, failures = idx.update_from_string(v1.read(), True)
             self.assertTrue(res)
             self.assertListEqual(failures, [])
 
         for fname in [
-            "spec.v2.yaml",
-            "translations/spec.v1.yaml",
-            "mod-defaults/spec.v1.yaml",
+            "yaml_specs/modulemd_stream_v2.yaml",
+            "yaml_specs/modulemd_translations_v1.yaml",
+            "yaml_specs/modulemd_defaults_v1.yaml",
```

Comment:
This would be a good time to add a test here that reads `"yaml_specs/modulemd_packager_v2.yaml"`. Although not in this loop...

Reply:
It cannot actually be read into a ModuleIndex because it has no `name` attribute to use as a lookup key. It must be read by `Modulemd.ModuleStream.read_*()` instead.

Reply:
I was aware of that. What I overlooked was that this is `module**index**.py`, so it doesn't belong here. Such a python test belongs in `module**stream**.py`--which is exactly what your added `test_packager_sanity()` does. Nevermind.  :)